src/Spice86.Core/Emulator/VM/Machine.cs

```diff
@@ -192,6 +194,16 @@ public class Machine : IDisposable {
         _dmaThread = new Thread(DmaLoop) {
             Name = "DMAThread"
         };
+        if(configuration.Ems) {
+            Ems = new(this, loggerService);
+        }
+        if(Ems is not null) {
```

Comment:
Why not just instantiating it and registering it in the same if for configuration?